src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -260,15 +316,53 @@ def main():
 
     # datasub = data[['dt', 'bw', 'tw', 'twcc', 'dx', 'n', 'ncc', 'cs', 's0']]
 
+    # STEP 5: Read (or set) QLateral Inputs
+    if showtiming:
+        start_time = time.time()
+    if verbose:
+        print("creating qlateral array ...")
+
+    # initialize qlateral dict
+    qlateral = {}
+
+    if qlat_input_folder:
+        qlat_files = glob.glob(qlat_input_folder + qlat_file_pattern_filter)
+        qlat_df = nhd_io.get_ql_from_wrf_hydro(
+            qlat_files=qlat_files,
+            index_col=qlat_file_index_col,
+            value_col=qlat_file_value_col,
+        )
+
+    elif qlat_input_file:
+        qlat_df = nhd_io.get_ql_from_csv(qlat_input_file)
+
+    else:
+        qlat_df = pd.DataFrame(
+            qlat_const, index=connections.keys(), columns=range(nts), dtype="float32"
```

Comment:
The qlf and qlw options produce dataframes that will need to be checked and trimmed/filled-with-zeros so they are the same size as this constant input qlateral that we've defined as the most basic input option.

Reply:
[28 rows x 144 columns] - working constant qlat size 
[374 rows x 24 columns] - broken command size

I'm going to go ahead and trim the rows and add a bunch of zeros to the columns then.

Reply:
This sizing issue seems to be the last remaining question. (Saw that the file was blackened ...)

Reply:
> trim the rows
The rows represent the segment ID (i.e., `connections`) and there are several dataframes which use that table and could be the source of an index to filter the qlat dataframe... 

> add a bunch of zeros
That's probably the right first step. The 24 hours in the current dataset will need to be subdivided into 300s timesteps, but let's get a test with zero padding to work first.

Reply:
@jhrehanoaa 
Please add an issue to implement time step adjustment for qlaterals in v02. Please also add an issue to validate the v02 computation against the Pocono_TEST2 test case.